### DISCUSSION TITLE
Need help with customization on cloud

### DISCUSSION THREAD

**Discussion by mconnect18** [appwrite/appwrite#5537] [Q&A] [answered]
Hi Appwrite Community,

I'm currently setting up an Appwrite installation on DigitalOcean using Docker Compose. I have a couple of questions and would appreciate any assistance or guidance you can provide:

How can I customize the .env file for Docker Compose to establish an external database connection? Specifically, I'd like to configure Appwrite to use a separate database hosted on another server.

I noticed that passwords in the .env file are encrypted. Could you please let me know what encryption key Appwrite uses for encrypting passwords in the .env file?

Any insights, tips, or examples related to these topics would be highly appreciated. Thank you in advance for your assistance!

**Comment by joeyouss** [comment, marked as answer]
Hi - currently you cannot change the environment variables on cloud, Currently on cloud, only connection to database hosted to us is possible. In future, we plan to improve it and add more features.

Coming to next question, what do you mean by passwords are encrypted? they are plain text - can you explain your question please?

Let me know if anymore help is needed.
